tests/shakedown/shakedown/dcos/command.py

```diff
@@ -210,7 +211,7 @@ def attached_cli():
 
     The CLI setup command should be idempotent. So it is save to call this method multiple times.
     """
-    cmd = 'cluster setup {} --no-check'.format(dcos_url())
+    cmd = 'cluster setup {} --no-check --username={} --password={}'.format(dcos_url(), dcos_username(), dcos_password())
```

Comment:
So, we had this around and it was working so far? 😮

Reply:
We should not require this since the newest DC/OS uses environment variables which we expose here https://github.com/mesosphere/marathon/blob/master/Jenkinsfile.strict.si#L28

Reply:
Well, it didn't work without it so ¯\_(ツ)_/¯ Maybe we're not on the _latest_ cli yet?